ui/app/templates/components/job-editor/read.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="boxed-section">
+    <div class="boxed-section-head">
+    Job Definition
+    <div class="pull-right" style="display: flex">
+        <div class="select-mode" data-test-toggle={{@data.view}}>
+        <button 
+            class="button is-small is-borderless {{if (eq @data.view "job-spec") "is-active"}}"
+            type="button"
+            {{on "click" @fns.onToggle}}
+        >
+            Job Spec
+        </button>
+        <button 
+            class="button is-small is-borderless {{if (eq @data.view "full-definition") "is-active"}}"
+            type="button"
+            {{on "click" @fns.onToggle}}
+            data-test-toggle-full
+        >
+            Full Definition
+        </button>
+        </div>
```

Comment:
This should be conditionally rendered because old jobs will not have this functionality. This wasn't addressed in the mocks so I will tackle this issue separately.